leveleditor/holidays/halloween.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name_longhand": "halloween_props",
+    "storage_files": [
+        "phase_4/dna/halloween_props_storage_TT.dna",
+        "phase_4/dna/halloween_props_storage_TT_sz",
+        "phase_5.5/dna/halloween_props_storage_estate.dna",
+        "phase_6/dna/halloween_props_storage_DD.dna",
+        "phase_6/dna/halloween_props_storage_GS.dna",
+        "phase_6/dna/halloween_props_storage_MM.dna",
+        "phase_6/dna/halloween_props_storage_OZ.dna",
+        "phase_8/dna/halloween_props_storage_BR.dna",
+        "phase_8/dna/halloween_props_storage_DG.dna",
+        "phase_8/dna/halloween_props_storage_DL.dna"
+    ]
+}
```

Comment:
Maybe just add a "holiday_storage_files" section to each hood json? and of course make the loader ok with there being none
like 
```json
"holiday_storage_files": {
   "halloween": [
        "phase_4/dna/halloween_props_storage_TT.dna",
        "phase_4/dna/halloween_props_storage_TT_sz"
     ],
   "winter": [
        "files"
    ]
}
```